app/models/page.rb

```diff
@@ -57,6 +57,10 @@ def meta_tags
     tag_names << plugin_names
   end
 
+  def meta_twitter_handle
+    Settings.meta_twitter_handles[language.code.to_sym]
```

Comment:
is there a reason to make this a method of the page? It has nothing to do with the page except the language